### PR TITLE
[Wallet] UI changes for CICO

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Change the CICO flow following the new design: https://www.figma.com/file/zt7aTQ5wuXycIwxq5oAmF9/Wallet--Refresh?node-id=10159%3A4

### Tested

Manually

### Related issues

- Part of #5424

### CODE REVIEW

File: packages/mobile/src/account/FiatExchange.tsx [hunk 1]
```diff
@@ -82,6 +78,12 @@ function FiatExchange() {
             </Text>
           </ListItem>
         )}
+        {regionCode === 'PH' && (
```
Comment (jeanregisser) [inline packages/mobile/src/account/FiatExchange.tsx]: Should we move this to a country feature flag? Or is this eventually gonna be dynamic from the merchants list?

**Review by jeanregisser — APPROVED** [review]
Looking good!

File: packages/mobile/src/account/FiatExchange.tsx (hunk 1 above)
Comment (gnardini) [reply packages/mobile/src/account/FiatExchange.tsx]: I can see us having different things in the `Spend` screen for different countries in the future, I'll move it to a country feature flag for now and we'll see I guess?

File: packages/mobile/src/config.ts [hunk 2]
```diff
@@ -106,32 +107,52 @@ export const MOONPAY_RATE_API = `https://api.moonpay.io/v3/currencies/celo/price
...
+]
+
+export const SPEND_MERCHANT_LINKS: SpendMerchant[] = [
+  {
+    name: 'Beam and Go',
+    link: 'https://valora.beamandgo.com/',
+  },
+  {
+    name: 'Merchant Map',
+    link: '',
```
Comment (annakaz) [inline packages/mobile/src/config.ts]:
```suggestion
    link: 'https://celo.org/experience/merchant/merchants-accepting-celo#map',
```

**Review by annakaz — APPROVED** [review]
Took one last look over, looks great and ready to merge

**Comment by Lss-Ankit** [comment]
Hi @tarikbellamine Verified task on latest test flight build v1.5.5 (34) and Android play store internal build v1.5.5 (1004294323) and updated the status in below sheet

Test plan sheet: https://docs.google.com/spreadsheets/d/1phHo7MUMQNLjuJOic1TlZwoHM2T0iecT07VLoKtiqMg/edit#gid=935380555

Existing issue:

#5840 

Please let me know if i need to verify thing else?

**Pull request merged**